src/base/bittorrent/session.cpp

```diff
@@ -301,7 +301,7 @@ Session::Session(QObject *parent)
     , m_isAddTorrentPaused(BITTORRENT_SESSION_KEY("AddTorrentPaused"), false)
     , m_isCreateTorrentSubfolder(BITTORRENT_SESSION_KEY("CreateTorrentSubfolder"), true)
     , m_isAppendExtensionEnabled(BITTORRENT_SESSION_KEY("AddExtensionToIncompleteFiles"), false)
-    , m_refreshInterval(BITTORRENT_SESSION_KEY("RefreshInterval"), 1500)
+    , m_UIUpdateInterval(BITTORRENT_SESSION_KEY("RefreshInterval"), 1500)
```

Comment:
I'm sorry, I won't even look beyond this place. Looks like you're creating an inappropriate relationship.
BitTorrent Session is Core component so it should know nothing about any UI etc. It can only report some events within itself.

Reply:
I think you overreacted, it is just a naming, I'll rename it to something else.

Reply:
>I think you overreacted, it is just a naming

I don't think "it is just a naming".